### PR TITLE
Fix travis S3 deplyment 

### LINKED ISSUE #6979: AWS update breaking `deploy-s3` task on Travis?
From Travis:
```
[0K$ npm run deploy-s3 -- -b cesium-dev -d cesium/$TRAVIS_BRANCH --confirm -c 'no-cache'

> cesium@1.48.0-additionalProjections28414 deploy-s3 /home/travis/build/AnalyticalGraphicsInc/cesium
> gulp deploy-s3 "-b" "cesium-dev" "-d" "cesium/additionalProjections" "--confirm" "-c" "no-cache"

[19:45:11] Using gulpfile ~/build/AnalyticalGraphicsInc/cesium/gulpfile.js
[19:45:11] Starting 'deploy-s3'...
[19:45:11] 'deploy-s3' errored after 18 ms
[19:45:11] TypeError: Cannot promisify an API that has normal methods with 'Async'-suffix

    See http://goo.gl/MqrFmX

    at checkValid (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/bluebird/js/release/promisify.js:54:27)
    at promisifiableMethods (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/bluebird/js/release/promisify.js:77:5)
    at promisifyAll (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/bluebird/js/release/promisify.js:241:9)
    at new Promise.promisifyAll (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/bluebird/js/release/promisify.js:311:12)
    at deployCesium (/home/travis/build/AnalyticalGraphicsInc/cesium/gulpfile.js:344:14)
    at Gulp.<anonymous> (/home/travis/build/AnalyticalGraphicsInc/cesium/gulpfile.js:315:9)
    at module.exports (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/orchestrator/index.js:134:8)
    at /home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/gulp/bin/gulp.js:129:20
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cesium@1.48.0-additionalProjections28414 deploy-s3: `gulp deploy-s3 "-b" "cesium-dev" "-d" "cesium/additionalProjections" "--confirm" "-c" "no-cache"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cesium@1.48.0-additionalProjections28414 deploy-s3 script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-08-30T19_45_11_871Z-debug.log

```

The line referenced in here seems to call the `aws-sdk`, which was updated very recently.

### PR DESCRIPTION
**Pull request by ggetz**
Fixes #6979

Use `.promise()` instead of `promisify` on the aws-sdk calls.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ggetz!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/fix-deploy/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: gulpfile.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ var Promise = require('bluebird');
 var requirejs = require('requirejs');
 var Karma = require('karma').Server;
 var yargs = require('yargs');
-var aws = require('aws-sdk');
+var S3 = require('aws-sdk/clients/s3');
```
Comment (mramato) [inline gulpfile.js]: You should never index into a node module like this.  require in `aws-sdk` like before and then use `AWS.S3`

**Comment by mramato** [comment]
Just the one comment.

**Comment by ggetz** [comment]
Thanks @mramato, updated!

**Pull request merged**